core-web/libs/data-access/src/lib/dot-license/dot-license.service.ts

```diff
@@ -124,10 +126,43 @@ export class DotLicenseService {
     }
 
     private getLicense(): Observable<DotLicense> {
+        if (this.license) return of(this.license);
+
         return this.coreWebService
             .requestView({
                 url: this.licenseURL
             })
-            .pipe(pluck('entity', 'config', 'license'));
+            .pipe(
+                pluck('entity', 'config', 'license'),
```

Comment:
Do we need a take(1)?

Reply:
It's not needed, `coreWebService` uses `HttpClient` to fetch and that service uses the `take(1)` under the hood